debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/structureformat/extensions/J9ObjectStructureFormatter.java

```diff
@@ -221,6 +221,9 @@ private void formatArrayObject(PrintStream out, J9ClassPointer localClazz, U8Poi
 				}
 				break;
 			case 'L':
+			/*[IF INLINE-TYPES] */
+			case 'Q':
+			/*[ENDIF] INLINE-TYPES */
```

Comment:
While you're here, can you please correct the indentation of the `break` on line 233.